hack/run-e2e-conformance-virtual-cluster.sh

```diff
@@ -364,5 +364,14 @@ hack/deploy-wait.sh
 
 if [ -z $SKIP_TEST ]; then
   echo "## run sriov e2e conformance tests"
-  SUITE=./test/conformance JUNIT_OUTPUT=`pwd`/k8s-artifacts hack/run-e2e-conformance.sh
+
+  if [[ -v TEST_REPORT_PATH ]]; then
+    JUNIT_OUTPUT="${TEST_REPORT_PATH}/conformance-test-report"
```

Comment:
is `pwd` not longer needed?

Reply:
Good catch: relative paths are relatives to the test suite entrypoint, i.e. `test/conformance/test_suite_test.go`